internal/docs/alert_rule_template.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package docs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type alertRuleTemplate struct {
+	Attributes struct {
+		Name        string
+		Description string
+	}
+}
+
+func renderAlertRuleTemplates(packageRoot string) (string, error) {
+	templatesDir := filepath.Join(packageRoot, "kibana", "alerting_rule_template")
+
+	if _, err := os.Stat(templatesDir); os.IsNotExist(err) {
+		// no template directory in the package, do nothing
+		return "", nil
+	}
+
+	var builder strings.Builder
```

Comment:
Can we add Alerts header?
`builder.WriteString("**Alert Rule Templates**\n\n")`.